session_1/hello_world/Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
```

Comment:
For session 1, please make sure you have one `<p>` in your about me and the text should match in `<h1>`